flink-ai-flow/ai_flow/store/db/db_model.py

```diff
@@ -124,13 +137,13 @@ class SqlModelVersionRelation(base):
 
     version = Column(String(255), primary_key=True)
     model_id = Column(BigInteger, ForeignKey('model_relation.uuid', onupdate='cascade'), primary_key=True)
-    workflow_execution_id = Column(BigInteger)
+    project_snapshot_id = Column(BigInteger, ForeignKey('project_snapshot.uuid', onupdate='cascade'))
     is_deleted = Column(String(256), default='False')
 
     UniqueConstraint(version, model_id)
```

Comment:
If you regards the model_id and version as composite primary key, this doesn't need to be constrainted unique.

Reply:
Sure, updated.